_posts/2019-04-17-cockpit-192.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Cockpit 192
+author: pitti
+date: 2019-04-17
+tags: cockpit linux
+slug: cockpit-192
+category: release
+summary: Cockpit with Machines and translation improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 192.
+
+### Machines: Auto-detect guest operating system
+
+When creating a new VM, the "OS Vendor" and "Operating System" fields are now
+set automatically after specifying the installation sources. This uses the
+libvirt [osinfo-detect](https://libosinfo.org/) utility.
```

Comment:
I think @KKoukiou mentioned this detection only works for a small subset of Linuxes — specifically RPM-based distributions. (And even then, possibly not _all_ RPM-based distros.)

Is this correct? Should we state something about this limitation?

It seems like a pretty big oversight if we cannot detect Debian, Ubuntu, Arch, Alpine, BSDs, and _especially_ Windows.

Reply:
Seems to be quite comprehensive:
```
❱❱❱ osinfo-detect http://releases.ubuntu.com/18.04.2/ubuntu-18.04.2-live-server-amd64.iso
Media is bootable.
Media is an installer for OS 'Ubuntu 18.04 LTS'
Media is live media for OS 'Ubuntu 18.04 LTS'

❱❱❱ osinfo-detect https://cdimage.debian.org/debian-cd/current/i386/iso-cd/debian-9.8.0-i386-netinst.iso
Media is bootable.
Media is an installer for OS 'Debian 9'
```

But not perfect indeed:
```
❱❱❱ osinfo-detect http://dl-cdn.alpinelinux.org/alpine/v3.9/releases/x86_64/alpine-standard-3.9.3-x86_64.isoMedia is bootable.
```
I'll reword it to mention "attempt".